content/projects/projectomes.html

```diff
@@ -84,8 +90,7 @@ <h4>Web Services</h4>
       {% endfor %}
     </div>
   </div>
-  
-  <h3>Explore</h3>
```

Comment:
If you are removing this then maybe move the dmri and fmri headers above the table elements so they render at the same size as the other sub headings